viz/visualize.py

```diff
@@ -127,9 +131,9 @@ def samples(self, size=(8, 8), filename='imgs/samples.png'):
         generated = self._decode_latents(prior_samples)
 
         if self.save_images:
-            save_image(generated.data, filename, nrow=size[1])
+            save_image(generated.data, filename, nrow=size[1], pad_value=10)
```

Comment:
the padding color will have to depend on the dataset. FOr mnist / dsprites it should be 1 (because background is 0) for the 2 others it should be 0 (because background is lighter/white). If you agree, then use a `get_background` function in `datasets` (you can do it the same way as `get_img_size`: i.e. just add a background class attribute to all images). 
Once you have background just use `pad_value=1-background`

+ I don't understand I thought that pad_value was the value to pad, i.e. between 0 and 1 , but you give 10. Yesterday I was saying to use the number of pixel to be 10 `padding=10`. What's happening here (i.e. how come does it work ??)

Reply:
I don't quite see where ```get_background``` is?

I am not sure why it works to be honest. It aught to work for padding_value=0 as well but it does not.

Reply:
I have taken the suggestion and implemented this in the latest commit